dotcom-rendering/src/components/InteractiveBlockComponent.importable.tsx

```diff
@@ -349,6 +350,12 @@ export const InteractiveBlockComponent = ({
 			? true
 			: false;
 
+	const isApps = renderingTarget === 'Apps';
+
+	const onTouchStart = () => getInteractionClient().disableArticleSwipe(true);
+
+	const onTouchEnd = () => getInteractionClient().disableArticleSwipe(false);
```

Comment:
Small performance suggestion.

I think these functions will be created on every render of this component even though they may not be used by the web rendering target. We could inline them in but again they will be created on every render of the component. [useCallback](https://react.dev/reference/react/useCallback) is an option but given the functions are idempotent and afaics don't rely on any state could they be moved outside of the component to the module level?